src/index.js

```diff
@@ -54,8 +54,8 @@ export default function mitt(all: EventHandlerMap) {
 		 * @memberOf mitt
 		 */
 		emit(type: string, evt: any) {
-			(all[type] || []).slice().map((handler) => { handler(evt); });
-			(all['*'] || []).slice().map((handler) => { handler(type, evt); });
+			(all[type] || []).map((handler) => { handler(evt); });
```

Comment:
I think that the intention was to cheaply create a copy of the original array to prevent any changes to it. 
Explanation can be found here: https://medium.com/daily-js-tips/using-slice-to-copy-an-array-f193a93b4a31
The only think I do not see is that the `.map` function does not mutate data so it should be safe to include your change

Reply:
That's my point exactly. I can see it can be valuable in many scenarios but it seems it's not the case.

Reply:
The one you mentioned is about IE9 lacking `Array#forEach()` support, my PR still keeps the `Array#map()` used previously, so there shouldn't be any issue with IE9.
Also, I'm only removing some code - not adding anything new so that afaik cannot break support in any case.

Reply:
@marcinlichwala Ahh, you're totally right.

I did a `git blame` to see why `slice` was introduced. I found this issue.
https://github.com/developit/mitt/issues/65

Reply:
@newyork-anthonyng Makes sense. `handler` call can modify the handlers array with `.off()` and the `Array.map()` will follow up on the changed (shifted) array therefore messing it up.